consent-ui/src/app/ais/common/consent-util.ts

```diff
@@ -1,22 +1,24 @@
 import {SessionService} from "../../common/session.service";
-import {AisConsent} from "./dto/ais-consent";
+import {AccountAccess, AisConsent, AisConsentToGrant} from "./dto/ais-consent";
```

Comment:
I guess all the models (DTOs) will be replaces by autogenerated models?

Reply:
Yes 
https://app.zenhub.com/workspaces/open-banking-gateway-5dd3b3daf010250001260675/issues/adorsys/open-banking-gateway/375